apis/filesystem/src/main/java/org/jclouds/filesystem/strategy/internal/FilesystemStorageStrategyImpl.java

```diff
@@ -745,28 +743,26 @@ protected String buildPathStartingFromBaseDir(String... pathTokens) {
    }
 
    /**
-    * Substitutes all the file separator occurrences in the path with a file separator for the
-    * current operative system
+    * Convert path to the current OS filesystem standard
     *
-    * @param pathToBeNormalized
+    * @param path
     * @return
     */
-   private static String normalize(String pathToBeNormalized) {
-      if (null != pathToBeNormalized && pathToBeNormalized.contains(BACK_SLASH)) {
-         if (!BACK_SLASH.equals(File.separator)) {
-            return pathToBeNormalized.replace(BACK_SLASH, File.separator);
-         }
+   private static String normalize(String path) {
+      if (null != path) {
```

Comment:
Windows has other restricted characters:

http://stackoverflow.com/a/31976060/2800111

We should not strive to make the filesystem blobstore work as similar on Linux as Windows by crippling the former.  Rather we should make the filesystem blobstore as closely match real blobstores as the environment allows.